base/client.jl

```diff
@@ -71,6 +71,12 @@ function repl_cmd(cmd, out)
         ENV["OLDPWD"] = new_oldpwd
         println(out, pwd())
     else
+        @windows_only begin
+            cmdbuf = IOBuffer()
+            print_cmdshell_escaped(cmdbuf, cmd.exec...)
+            cmd.exec = ["cmd"; "/s /c \"$(bytestring(cmdbuf.data))\""]
```

Comment:
I'm not so sure about this. Neither cmd.exe nor powershell.exe reside directly under WINDIR on my system:
```
c:\windows\system32\cmd.exe
c:\windows\System32\WindowsPowerShell\v1.0\powershell.exe
```
Relying on Windows path settings seems like a more robust choice.